202002_noise/noise.toml

```diff
@@ -0,0 +1,29 @@
+
+tag = "noise"
+
+[ other_components ]
+
+    [ other_components.noise ]
+
+    class = "mapsims.SONoiseSimulator"
```

Comment:
@mhasself can you please review the noise parameters? In particular I chose `rolloff_ell` of 50